src/ActionUnit.h

```diff
@@ -45,7 +45,7 @@ class ActionUnit
     friend class XMLimport;
 
 public:
-                          ActionUnit( Host * pHost ) : mpHost(pHost), mMaxID(0) {;}
+                          ActionUnit( Host * pHost ) : mpHost(pHost), mMaxID(0), mpToolBar(), mpEasyButtonBar(), mModuleMember() {;}
```

Comment:
And yes, value-initialization means that pointers get nullptr and booleans get false.